docs/debug/warmboot_reset.md

```diff
@@ -0,0 +1,184 @@
+Doubled sign of life
+====================
+
+Problem description
+-------------------
+
+Sometimes sign of life happens twice. It is caused by reset of platform during
+warmboot (doing `rte_ctrl -pon` when the platform is in S5). Problem does
+not occur when doing coldboot or reboot.
+
+Initial ideas
+-------------
+
+[AGESA specification](https://cloud.3mdeb.com/index.php/apps/files/?dir=/pcengines/docs&fileid=6969#pdfviewer)
+(page 41 and followings) in operational overview says:
```

Comment:
@krystian-hebel let's not use internal cloud links in public respos. To access that document users need login to cloud. If documentation is publically available then please link to public server, if not please mentione it is not publically avaialable. Either way please note document version.

Reply:
done